### PR TITLE
Clevo: usb stack enable

### PR DESCRIPTION
**Pull request by Stojek139808**

### CODE REVIEW

File: payloads/external/Makefile.inc [hunk 1]
```diff
@@ -207,6 +207,7 @@ $(obj)/UEFIPAYLOAD.fd edk2: $(DOTCONFIG) $(IPXE_EFI)
 		CONFIG_EDK2_DASHARO_POWER_CONFIG=$(CONFIG_EDK2_DASHARO_POWER_CONFIG) \
 		CONFIG_EDK2_DASHARO_POWER_CONFIG_FAN_CURVE=$(CONFIG_EDK2_DASHARO_POWER_CONFIG_FAN_CURVE) \
 		CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE=$(CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE) \
+		CONFIG_EDK2_DASHARO_USB_CONFIG=$(CONFIG_EDK2_DASHARO_USB_CONFIG) \
```
Comment (miczyg1) [inline payloads/external/Makefile.inc:210]: This line is already present above: https://github.com/Dasharo/coreboot/pull/346/files#diff-e9897668de3b042db9e1790c4ab68359db4c1a7dc392142e8e8b3725b00c14c7R204

File: payloads/external/edk2/Kconfig [hunk 2]
```diff
@@ -311,6 +311,13 @@ config EDK2_DASHARO_POWER_CONFIG_FAN_CURVE
 
 endif
 
+config EDK2_DASHARO_USB_CONFIG
```
Comment (miczyg1) [inline payloads/external/edk2/Kconfig:314]: ALreayd present in line https://github.com/Dasharo/coreboot/blob/2344a7c8fecc38bfdecd60f05727d463c8bcfad2/payloads/external/edk2/Kconfig#L278

File: payloads/external/edk2/Makefile [hunk 3]
```diff
@@ -137,6 +137,10 @@ endif
 ifneq ($(CONFIG_EDK2_DISABLE_MTRR_PROGRAMMING),y)
 BUILD_STR += -D DISABLE_MTRR_PROGRAMMING=FALSE
 endif
+# DASHARO_USB_CONFIG	= FALSE
```
Comment (miczyg1) [inline payloads/external/edk2/Makefile:140]: Already present in line: https://github.com/Dasharo/coreboot/blob/2344a7c8fecc38bfdecd60f05727d463c8bcfad2/payloads/external/edk2/Makefile#L155

**Review by miczyg1 — CHANGES_REQUESTED** [review]
Just close this PR. I don't know what it was supposed to do...

File: payloads/external/Makefile.inc (hunk 1 above)
Comment (Stojek139808) [reply payloads/external/Makefile.inc]: This was pushed by accident and no longer present. Now this PR has correct changes. Closing.

File: payloads/external/edk2/Kconfig (hunk 2 above)
Comment (Stojek139808) [reply payloads/external/edk2/Kconfig]: as in comment above

File: payloads/external/edk2/Makefile (hunk 3 above)
Comment (Stojek139808) [reply payloads/external/edk2/Makefile]: as above

File: configs/config.novacustom_nv4x_adl_full [hunk 4]
```diff
@@ -44,14 +44,14 @@ CONFIG_EDK2_DASHARO_INTEL_ME_OPTIONS=y
 CONFIG_EDK2_DASHARO_POWER_CONFIG=y
 CONFIG_EDK2_DASHARO_POWER_CONFIG_FAN_CURVE=y
 CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE=y
+CONFIG_EDK2_DASHARO_USB_CONFIG=y
 CONFIG_EDK2_BOOT_MENU_KEY=0x0011
 CONFIG_EDK2_SETUP_MENU_KEY=0x000C
 # CONFIG_EDK2_SECURE_BOOT_DEFAULT_ENABLE is not set
-CONFIG_EDK2_CUSTOM_BUILD_PARAMS="-D VARIABLE_SUPPORT=SMMSTORE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdSkipPs2Detect=TRUE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdPrintSolStrings=TRUE"
+CONFIG_EDK2_CUSTOM_BUILD_PARAMS="-D VARIABLE_SUPPORT=SMMSTORE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdSkipPs2Detect=TRUE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdPrintSolStrings=TRUE -D VARIABLE_SUPPORT=SMMSTORE -D IOMMU_ENABLE=TRUE"
```
Comment (miczyg1) [inline configs/config.novacustom_nv4x_adl_full:51]:
```suggestion
CONFIG_EDK2_CUSTOM_BUILD_PARAMS="-D VARIABLE_SUPPORT=SMMSTORE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdSkipPs2Detect=TRUE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdPrintSolStrings=TRUE -D IOMMU_ENABLE=TRUE"
```

a/a

File: configs/config.novacustom_nv4x_tgl_full [hunk 5]
```diff
@@ -44,13 +44,13 @@ CONFIG_EDK2_DASHARO_POWER_CONFIG_FAN_CURVE=y
 CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE=y
 CONFIG_EDK2_BOOT_MENU_KEY=0x0011
 CONFIG_EDK2_SETUP_MENU_KEY=0x000C
+CONFIG_EDK2_DASHARO_USB_CONFIG=y
 CONFIG_DRIVERS_EFI_VARIABLE_STORE=y
 # CONFIG_EDK2_SECURE_BOOT_DEFAULT_ENABLE is not set
-CONFIG_EDK2_CUSTOM_BUILD_PARAMS="-D VARIABLE_SUPPORT=SMMSTORE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdSkipPs2Detect=TRUE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdPrintSolStrings=TRUE"
+CONFIG_EDK2_CUSTOM_BUILD_PARAMS="-D VARIABLE_SUPPORT=SMMSTORE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdSkipPs2Detect=TRUE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdPrintSolStrings=TRUE -D VARIABLE_SUPPORT=SMMSTORE -D IOMMU_ENABLE=TRUE"
```
Comment (miczyg1) [inline configs/config.novacustom_nv4x_tgl_full:50]:
```suggestion
CONFIG_EDK2_CUSTOM_BUILD_PARAMS="-D VARIABLE_SUPPORT=SMMSTORE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdSkipPs2Detect=TRUE --pcd gUefiPayloadPkgTokenSpaceGuid.PcdPrintSolStrings=TRUE -D IOMMU_ENABLE=TRUE"
```

a/a

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**